### PR TITLE
Update feedback page to allow users to upload profile picture with comment

### PR DESCRIPTION
**Pull request by dbuford**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    commentEntity.setProperty("name", words[0]);
+    commentEntity.setProperty("email", words[1]);
+    commentEntity.setProperty("timestamp", timestamp);
+    commentEntity.setProperty("image", imageUrl);
+   
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(commentEntity);
+
+
+    PrintWriter out = response.getWriter();
```
Comment (honkwan) [inline portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java:96]: How useful is this page?

When I comment on other websites like YouTube, Facebook etc, there isn't a thank you page and I don't have to click an extra button to go back.

Up to you.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -115,19 +104,171 @@ function addRandomGreeting() {
...
+let map;
+
+/* Editable marker that displays when a user clicks in the map. */
+let editMarker;
+
+/** Creates a map that allows users to add markers. */
+function createMap() {
+  map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 38.5949, lng: -94.8923}, zoom: 4});
```
Comment (honkwan) [inline portfolio/src/main/webapp/script.js:127]: Add a comment about where is this location

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -115,19 +104,171 @@ function addRandomGreeting() {
...
+  map.addListener('bounds_changed', function() {
+    searchBox.setBounds(map.getBounds());
+  });
+
+  // Listen for the event fired when the user selects a prediction and retrieve
+  // more details for that place.
+  searchBox.addListener('places_changed', function() {
+    var places = searchBox.getPlaces();
+
+    if (places.length == 0) {
```
Comment (honkwan) [inline portfolio/src/main/webapp/script.js:143]: is there a .empty() function instead?

**Review by honkwan — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java (hunk 1 above)
Comment (dbuford) [reply portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java:96]: Yes, I think that sounds like a better idea

**Pull request merged**